File: py i cpp/wej-wyj.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
	char imie[20];
    char nazwisko[20];
    
    cout << "Ładnie proszę, podaj swoje imię: ";
    cin >> imie;
    cout << "Ładnie proszę, podaj swoje nazwisko: ";
    cin >> nazwisko;
    cout << "Witaj" << " " << imie << " " << nazwisko << "! :)" << endl;
    
	return 0;
}

File: py i cpp/wej-wyj.py
def main(args):
    a = input("Podaj imię: ")
    b = input("Podaj nazwisko: ")
    
    print("Witaj", a, b + "! :)")
    
    return 0



if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
